src/GUI/RevEng.Core/ReplacingCandidateNamingService.cs

```diff
@@ -96,6 +101,17 @@ public override string GenerateCandidateIdentifier(DatabaseColumn originalColumn
                 }
             }
 
+            if (!string.IsNullOrEmpty(schema.ColumnRegexPattern) && schema.ColumnPatternReplaceWith != null)
+            {
+                string newColumnName = Regex.Replace(originalColumn.Name, schema.ColumnRegexPattern, schema.ColumnPatternReplaceWith);
```

Comment:
Please add options with a timeout

Reply:
Did add a timeout to regex rename method